src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/CreateCaseValidator.java

```diff
@@ -66,18 +70,50 @@ public CreateCaseValidator() {
     }
 
     public Validation<Seq<String>, ExceptionRecord> getValidation(CaseDetails caseDetails) {
-        return Validation
-            .combine(
-                hasCaseTypeId(caseDetails),
-                hasPoBox(caseDetails),
-                hasJurisdiction(caseDetails),
-                hasJourneyClassification(caseDetails),
-                hasDateField(caseDetails, "deliveryDate"),
-                hasDateField(caseDetails, "openingDate"),
-                getScannedDocuments(caseDetails),
-                getOcrDataFields(caseDetails)
-            )
-            .ap(ExceptionRecord::new);
+
+        Validation<String, String> hasCaseTypeId = hasCaseTypeId(caseDetails);
```

Comment:
Something just to consider: maybe we should name these variables `{fieldName}Validation` instead of `has{FieldName}`?

Reply:
agree, renamed variable names